src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -1483,6 +1483,38 @@ void Decoder::DecodeCRType(Instruction* instr) {
       UNSUPPORTED_RISCV();
   }
 }
+
+void Decoder::DecodeCAType(Instruction* instr) {
+  switch (instr->RvcFunct6Value()) {
+    case 0b100011:
+      switch (instr->RvcFunctValue()) {
+       case 0b00: //c.sub
+         if (instr->RvcRdValue() != 0 && instr->RvcRs2Value() != 0)
+           Format(instr, "sub       'Crs1s, 'Crs1s, 'Crs2s");
+         else
+           UNSUPPORTED_RISCV();
+	 break;
```

Comment:
Indentation does not look right here. Did you run the clang formator (please follow instructions [here](https://github.com/v8-riscv/v8/wiki/VSCode-Setup#code-formatting))?